modules/45_carbonprice/exogenous/datainput.gms

```diff
@@ -21,7 +21,7 @@ pm_taxCO2eq(ttot,regi)$(ttot.val ge 2005) = p45_tau_co2_tax(ttot,regi);
 
 else
   pm_taxCO2eq(ttot,regi)$(ttot.val ge 2005) = 0;
-abort "Error: Please set cm_emiscen to 9";
+abort "Error: Please set cm_emiscen to 9 and cm_iterative_target_adj to 0";
```

Comment:
Depending on the need of the group it might be good to turn this into a warning that can be ignored if a different `cm_iterative_target_adj` was intentionally chosen. But maybe there is no use-case for this.

Reply:
ok, convinced! Go ahead